### DISCUSSION TITLE
How to customize a PDF (LaTeX) export template?

### DISCUSSION THREAD

**Discussion by eraus** [Zettlr/Zettlr#2849] [Q&A]
Hello folks,

I am a Windows user. I am trying to customize the layout of the export of PDF (LaTeX). I use LaTeX on and off, but I cannot find where to start. Specifically, I have the following three questions:

1. Where (the path and file name) can I find the default PDF (LaTeX) template? 
2. Where can I find the documentation about what to change in the template to change the layout of the exported pdf file?
3. How do I set up to use my newly created template?

Please help. Please be specific. 

Thanks a lot,
JHL

**Comment by nathanlesage** [comment]
1. There is no filename, since the LaTeX template is baked into the Pandoc binary. However, you can find the template on GitHub [here](https://github.com/jgm/pandoc/blob/master/data/templates/default.latex). You can download and adapt it, and then point Zettlr to use that one by adding a `template: /path/to/your/custom/template.tex` into the LaTeX PDF defaults file
2. That documentation is unfortunately a tad lacking on the Pandoc side of things. In general, you can adapt a lot by using YAML frontmatter or defaults variables [as outlined here](https://docs.zettlr.com/en/core/assets-manager/#how-to-adapt-the-pdf-preferences). All possible variables that the default template accepts (besides the ones outlined in the Zettlr docs) are described [here](https://pandoc.org/MANUAL.html#variables-for-latex). You can also add your own, custom ones by adapting the template itself.
3. See above answer to 1: Use the `template` var.

**Comment by sebs** [comment]
looks like this last explainer could go into the docs? Was missing the info there.
